tendrl/node_agent/flows/configure_collectd/__init__.py

```diff
@@ -0,0 +1,50 @@
+import socket
```

Comment:
https://github.com/Tendrl/commons/pull/661#pullrequestreview-53777920

This flow should be merged into the atom mentioned at above link